plugin.json

```diff
@@ -2,7 +2,7 @@
     "id": "com.mattermost.plugin-todo",
     "name": "To Do Plugin",
     "description": "This plugin makes it easy to keep track of to do issues and get daily reminders.",
-    "version": "0.1.0",
+    "version": "0.2.0",
```

Comment:
Why is the version bump part of this PR?

Reply:
It should not be. I inadvertently branched from the local version bump branch instead of master.  Removed the plugin bumps